ui/desktop/electron-app/src/services/cache-daemon-manager.js

```diff
@@ -23,15 +24,21 @@ class CacheDaemonManager {
 
   /**
    * Checks the status of the cache daemon.
-   * @returns {string}
+   * @returns {Promise}
    */
   status() {
     const daemonStatusCommand = ['cache', 'status', '-format=json'];
-    const { stdout } = spawnSync(daemonStatusCommand);
+    const { stdout, stderr } = spawnSync(daemonStatusCommand);
+
+    const parsedResponse = jsonify(stdout);
+    this.#socketPath = parsedResponse?.item?.socket_address;
+
+    if (parsedResponse?.status_code === 200) {
+      return parsedResponse.item;
+    }
 
-    const status = jsonify(stdout);
-    this.#socketPath = status?.item?.socket_address;
-    return status;
+    log.warn('Cache Daemon Status:', stderr);
```

Comment:
non-blocking: it looks like we will only know the version of the daemon if there is an error during this cmd. Im wondering if its possible to check the status at the very beginning along with the boundary cli version? Or perhaps knowing that info is not rly necessary ?

Reply:
Maybe I should've specified this is meant for logging errors only, I actually do log the cache daemon status which includes the version [here](https://github.com/hashicorp/boundary-ui/pull/2391/files#diff-1261de05929d4dcb2a372b5d2587fa51007aebfff1d9fa8abd8f9df3e9ac297fR59).

It looks like this:
<img width="1004" alt="image" src="https://github.com/user-attachments/assets/947d7218-1ac9-4450-816a-d88faf267cda">